### DISCUSSION TITLE
v0.4 and 0.5 compatibility for websites and users

### DISCUSSION THREAD

**Discussion by d47081** [yggdrasil-network/yggdrasil-go#1057] [Q&A]
I have few web-services, one of them is the search engine that crawling yggdrasil websites and running on 0.4 protocol 

So after the 0.5 release, will those websites not available for crawler anymore? What to do with the index collected.
Plus maintain public peer node, so what to do - run few separated nodes or upgrade to 0.5 and it will be compatible with connections from 0.4

Saw #905 but won't to mix my subject with the the roadmap discussions

**Comment by neilalexander** [comment]
Version 0.4 and 0.5 are not compatible, so once you upgrade to 0.5, any nodes still running 0.4 will become unreachable.

However, the IP addresses will not change between 0.4 and 0.5, so as people upgrade their nodes, those services will become reachable again.

**Reply by d47081** [reply]
Well, thanks for info 

some part of websites will get down so could be accessible through index snaps history until they wake up.
